packages/trader/src/sass/app/modules/trading-mobile.scss

```diff
@@ -388,6 +388,9 @@
 
             .contract-type-widget {
                 flex: 1;
+                display: flex;
+                justify-content: flex-start;
```

Comment:
```suggestion
```
@kate-deriv flex-start is a default value, can be omitted )

Reply:
Will do, thank you) Forgot about that